### ISSUE TITLE
Lat/Lon within Apollo Navigation Landmarks.mkr should be updated

### ISSUE THREAD

**Issue by Aeasala** [dseagrav/NASSP#1117]
When visual markers have been enabled within orbiter, the lat/lon of landmarks defined in this file do not match up with the actual landing sites showcased in documents.

For example, CP1 thru CP3 of Apollo 8 are many, many miles off.

Correcting the table to the lat/long shown in **Table V** (p. 21) of _Lunar Landmark Locations – Apollo 8, 10, 11, and 12 missions_ resolved their locations.

It is believed that the current values may have been derived from **Table IV** of the mentioned document, which are position 'solutions' for the tracking sequence.

**Comment by Aeasala** [comment]
I might be mistaken about the last note.
Either way, example of current selected values:
-162.700 -5.250        : CP1 504 (Apollo 8)
+155.100 -10.200     : CP2 526 (Apollo 8)
+95.9 -9.1              	: CP3 334 (Apollo 8)

From Table V:
-158.0462 -6.3079        : CP1 504 (Apollo 8)
+163.2463 -9.7036     : CP2 526 (Apollo 8)
+96.8915 -8.8990   	: CP3 334 (Apollo 8)

**Comment by indy91** [comment]
I will go through the coordinates some time and make necessary corrections. Thanks!

**Comment by indy91** [comment]
It's interesting. The coordinates we have been using are from the flight plan, so theoretically they should be good. We don't have the full Apollo 8 Lunar Landmark Maps document available but I found one page and the coordinates differ from the flight plan:

Flight Plan:

![image](https://github.com/orbiternassp/NASSP/assets/13571607/e52f2287-fad6-4481-a587-b370071a3ee6)

Landmark Maps:

![image](https://github.com/orbiternassp/NASSP/assets/13571607/f857672c-8acf-4919-a717-531e09d21247)

So our values should definitely be changed. Maybe not necessarily to the ones that got used on the actual mission, as the exact coordinates were more of a suggestion than exact. But we should definitely use the coordinates that Jim Lovell intended to use before the mission. I'll do some more research!

**Comment by Aeasala** [comment]
Yeah, not sure about how it should go forward, but the document I listed does provide all of the landmark coordinates to a T, geographically.  At least when comparing the visual references provided in the document for each landmark, there's pretty much zero error.

I'm assuming this .mkr file is only for visual aid if markers are enabled, and not actually referenced behind the scenes as NASSP does its thing.  If that's not the case, I like the idea in your last note.

**Comment by eddievhfan1984** [comment]
IIRC, the original .mkr files took into account that Orbiter 2016 had a flat mesh for the Moon, so shooting sights on landmarks had to account for that. Does the format also include altitude placement as well?

**Comment by Aeasala** [comment]
From what I can tell, I can't find any .mkr files -- vanilla to the base game, or not -- which have anything beyond lat/lon and a label for each entry.  So I would assume no.

**Comment by indy91** [comment]
Hey @eddievhfan1984 good to hear from you!

Orbiter 2016 introduced a new marker format which can take the elevation into account and which I have implemented for Earth landmarks. But I really dislike the way this has to be done and I am hesitant to implement it for the lunar landmarks. You basically have to change a file that comes with Orbiter and the rendering of the marker is tied to rendering of Orbiter textures itself. So it's quite inconsistent if they even show up! The best long term solution would probably still be this newer format, I might try it again some time.

**Comment by rcflyinghokie** [comment]
#1304

**Comment by ThatRedMelon** [comment]
With the newest release, #1389 has been merged, which matches up the lunar landmark makers pretty much exactly to the position from the landmark maps, so you can be sure you're marking on the correct spot. Some of these landmarks' flight plan positions are quite far away from their actual position (especially on the earlier missions like Apollo 8 or 10), I'll put a table of some of them below, but this is nowhere near exhaustive:

|Landmark|Flight Plan Position|Orbiter Position|
|:-:|:-:|:-:|
|Apollo 8 CP-1| 5.25S<br>162.7W |5.993S<br>162.1855W |
|Apollo 10 F-1| 1.6N<br>86.88E|1.926N<br>88.256E|

I'll close this issue, but if you see any issues feel free to comment!
